kb/how-to/add-test-generation.md

```diff
@@ -0,0 +1,329 @@
+# How to systematically generate test data
+
+When adding new test functions or improving coverage of existing tests in Mbed TLS, a large number of test cases may need to be added. This can be time-consuming and error-prone when adding manually. Mbed TLS includes a Python framework which can be used to systematically generate test case data from input values. The framework is flexible, to support a variety of test functions, and uses inheritance to reduce code repetition in similar test functions.
```

Comment:
I personally find long lines hard to read, even when the editor wraps them. Typically I find 90-100 characters a good maximum.

In addition, I also tend to start a new line when there's a new sentence - when this isn't done, a later PR changing the text will often cause reformatting of all the following lines, which makes diffs harder to read than they need to be.

Reply:
This is definitely an oversight, I'll add a commit to split the lines up.